nautobot_design_builder/design.py

```diff
@@ -20,10 +22,13 @@
 
 from nautobot_design_builder import errors
 from nautobot_design_builder import ext
-from nautobot_design_builder.logging import LoggingMixin
+from nautobot_design_builder.logging import LoggingMixin, get_logger
 from nautobot_design_builder.fields import field_factory, OneToOneField, ManyToOneField
 from nautobot_design_builder import models
+from nautobot_design_builder.constants import NAUTOBOT_ID
 from nautobot_design_builder.util import nautobot_version
+from nautobot_design_builder.recursive import inject_nautobot_uuids, get_object_identifier
+
 
 if nautobot_version < "2.0.0":
```

Comment:
I realize this got added in a previous PR, but I just noticed it... I don't think it is necessary to have the local version of `serialize_object_v2`. It is available in Nautobot in `nautobot.utilities.utils` so we just need an if/else with Nautobot version and two different imports

Reply:
If we don't overwrite it, how can we overcome the limitation in Nautobot 1.6.x for the `Status` serializer? (it has not the UUID in the serialization)